package/MDAnalysis/core/universe.py

```diff
@@ -311,9 +311,15 @@ class Universe(object):
     bonds, angles, dihedrals
         principal ConnectivityGroups for each connectivity type
 
+
     .. versionchanged:: 1.0.0
         Universe() now raises an error. Use Universe(None) or :func:`Universe.empty()` instead.
         Removed instant selectors.
+
+    .. versionchanged:: 2.0.0
+        Universe now can be (un)pickled.
+        Topology, trajectory and anchor_name are reserved upon unpickle.
```

Comment:
Are these attribute names? (i.e. `universe.topology`)? If so, could you uncapitalise "Topology" and also clarify a bit -- or even more clarification if I misinterpreted this note.